hpx/util/thread_aware_timer.hpp

```diff
@@ -50,7 +50,7 @@ namespace hpx { namespace util
 
         std::int64_t elapsed_microseconds() const
         {
-            return std::int64_t((take_time_stamp() - start_time_) * 1e-3);
+            return std::int64_t((take_time_stamp() - start_time_) * 1000);
```

Comment:
This change is not correct. The code should divide by `1000`, not multiply.

Reply:
Excellent catch, thanks!